composer.json

```diff
@@ -9,5 +9,17 @@
             "email": "hello@humanmade.com"
         }
     ],
-    "require": {}
+    "require": {
+        "humanmade/wp-seo": "0.14.0",
```

Comment:
We could do with agreeing on what upstream version contraints we want to use. I'm for `exact`, or `~1.0.0` (eg) as a default. Thoughts also @rmccue ?

Reply:
hm, I'd prefer exact - some 3rd party plugins can't be relied on to follow semver properly and it'll avoid surprises. Like we could potentially do it for our own plugins we control but I'm leaning towards exact

Reply:
👍 created https://github.com/humanmade/platform-dev/issues/313 to discuss